extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerTreeDataProvider.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { ExtensionNodeType, TreeItem } from 'azdata';
+import { ExtensionNodeType, TreeItem, connection } from 'azdata';
```

Comment:
This can be removed from here.

Reply:
Not just a nit - compilation failing problem 😄